app/controllers/comments_controller.rb

```diff
@@ -28,10 +30,10 @@ def update
   end
 
   def destroy
-    @comment = current_user.comments.find(params[:id])
+    @comment = Comment.find(params[:id])
```

Comment:
put in a help request for this -- I think the at admin part needs to go in the admin controller for comments